stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/model/AnnotatedClass.java

```diff
@@ -67,7 +66,7 @@
 
         FieldOption fieldOption = useStag != null ? useStag.value() : null;
         if (fieldOption == null) {
-            useStag = ElementUtils.findAnnotation(UseStag.class, element);
+            useStag = element.getAnnotation(UseStag.class);
```

Comment:
cc @mcumings I think this is okay to remove now, do you agree? The tests and build are all passing with their removal